index.html

```diff
@@ -2395,7 +2396,13 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
           response message for the primary response.</p><table class="def"><thead><tr><th><a>Vocabulary term</a></th><th>Description</th><th>Assignment</th><th>Type</th></tr></thead><tbody><tr class="rfc2119-table-assertion" id="td-vocab-contentType--ExpectedResponse"><td><code>contentType</code></td><td>Assign a content type based on a media type
                 (e.g., <code>text/plain</code>) and potential
                 parameters (e.g., <code>charset=utf-8</code>) for
-                the media type [<cite><a class="bibref" data-link-type="biblio" href="#bib-rfc2046" title="Multipurpose Internet Mail Extensions (MIME) Part Two: Media Types">RFC2046</a></cite>].</td><td>mandatory</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr></tbody></table></section>
+                the media type [<cite><a class="bibref" data-link-type="biblio" href="#bib-rfc2046" title="Multipurpose Internet Mail Extensions (MIME) Part Two: Media Types">RFC2046</a></cite>].</td><td>mandatory</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr>
+<tr class="rfc2119-table-assertion" id="td-vocab-schema--ExpectedResponse"><td><code>schema</code></td><td>Used to define the output data schema 
```

Comment:
This is the same text as for the "AdditionalExpectedResponse".
What is the difference between ExpectedResponse and AdditionalExpectedResponse?
Why does only the AdditionalExpectedResponse have a status code?
What does "default" mean for these two types?

Reply:
> This is the same text as for the "AdditionalExpectedResponse".

Yes, it the same intention.

> What is the difference between ExpectedResponse and AdditionalExpectedResponse?

ExpectedResponse can be defined only once. AdditionalExpectedResponse can be defined as array and may be need to define, e.g., different error responses messages (I think this was also requested by you).

>  Why does only the AdditionalExpectedResponse have a status code?

Adding this to ExpectedResponse would violate backwards compatibility.

> What does "default" mean for these two types?

See https://w3c.github.io/wot-thing-description/#sec-default-values